source/extensions/transport_sockets/tls/context_impl.cc

```diff
@@ -1112,6 +1112,11 @@ ServerContextImpl::selectTlsContext(const SSL_CLIENT_HELLO* ssl_client_hello) {
     break;
   }
 
+  // Apply the selected context. This must be done before OCSP stapling below
+  // since applying the context can remove the previously-set OCSP response.
+  RELEASE_ASSERT(SSL_set_SSL_CTX(ssl_client_hello->ssl, selected_ctx->ssl_ctx_.get()) != nullptr,
```

Comment:
Very small nit! Sorry about this.

Can you mention that this will only return NULL if memory allocation fails? (To justify the RELEASE_ASSERT to another reader).

Reply:
I will add that comment basically verbatim (note this is the same code as before just moved)

Reply:
Thanks!